lib/dor/workflow_exception.rb

```diff
@@ -0,0 +1,4 @@
+module Dor
+  class WorkflowException < ::RuntimeError
+  end
+end
```

Comment:
Oh, I was looking for something more substantial here.  I don't see how this puppy is a proxy for handling `RestClient` exceptions.